README.md

```diff
@@ -213,6 +213,13 @@ Optionally, you can also pass the template directory to which you want to instal
 $ sh -c "$(curl -fsSL https://r.viktoradam.net/githooks)" -- --use-core-hookspath --template-dir /home/public/.githooks
 ```
 
+If you want to install from another repository (e.g. from your own fork), you can specify the update repository url 
+as well as the branch name (default: `master`) when installing with:
+```shell
```

Comment:
Missing new line before the code block and missing `$` before the shell command. I'll add these directly to master after merging.

Reply:
thx